beacon-chain/sync/data_columns_sampling_test.go

```diff
@@ -126,10 +154,133 @@ func TestIncrementalDAS(t *testing.T) {
 		StateRoot:  emptyRoot[:],
 		BodyRoot:   emptyRoot[:],
 	}
-
 	emptyHeaderRoot, err := emptyHeader.HashTreeRoot()
 	require.NoError(t, err)
 
+	p2pSvc := p2ptest.NewTestP2P(t)
+	chainSvc, clock := defaultMockChain(t)
+
+	test := &dataSamplerTest{
+		ctx:        context.Background(),
+		root:       emptyRoot,
+		header:     emptyHeader,
+		headerRoot: emptyHeaderRoot,
+		p2pSvc:     p2pSvc,
+		peers:      []*p2ptest.TestP2P{},
+		ctxMap:     map[[4]byte]int{{245, 165, 253, 66}: version.Deneb},
+		chainSvc:   chainSvc,
+	}
+	sampler := newDataColumnSampler1D(p2pSvc, clock, test.ctxMap, nil)
+
+	return test, sampler
+}
+
+func TestDataColumnSampler1D_PeerManagement(t *testing.T) {
+	test, sampler := setupDefaultDataColumnSamplerTest(t)
```

Comment:
Is it possible to run this test with `custodyRequirement > 1`?

Reply:
added